pyproject.toml

```diff
@@ -52,7 +52,7 @@ requires = [
     "numpy~=1.21.2; python_version<'3.11'",
     "numpy~=1.23.3; python_version>='3.11' and python_version<'3.12'",
     "numpy~=1.26.0; python_version>='3.12'",
-    "pybind11~=2.11.1",
+    "nanobind~=1.8.0",
```

Comment:
Hang on. Are we actually using nanobind in any way, here?

The extension is a plain Python C extension these days.

Reply:
Hmm, true let me try just removing this.